tests/unit/specs/components/staking/Undelegations.spec.js

```diff
@@ -0,0 +1,36 @@
+import { shallowMount } from "@vue/test-utils"
+import Undelegations from "staking/Undelegations"
+import validators from "../../store/json/validators.js"
+import { flatOrderedTransactionList } from "../../store/json/txs"
+
+describe(`Undelegations`, () => {
+  let wrapper, $store
+  const getters = {
+    yourValidators: validators[0],
+    flatOrderedTransactionList
+  }
+
+  const state = {
+    session: { signedIn: true }
+  }
+
+  beforeEach(() => {
+    $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      state,
+      getters: JSON.parse(JSON.stringify(getters)) // clone so we don't overwrite by accident
```

Comment:
Lodash deepClone would be better. Also JSON parse won't handle functions right? Like `jest.fn()` that we might use.